.codecov.yml

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 # ----------------------------------------------------------------------------
-# Copyright (C) 2013-2019 British Crown (Met Office) & Contributors.
+# Copyright (C) British Crown (Met Office) & Contributors.
 #
```

Comment:
Am I right that we are no longer including the year?

Reply:
I think so. Only in one place now I think (License file? Can't recall)

Reply:
Thanks for doing this, very observant.

You can follow the model of Cylc Flow, just add a line in the `README.md`.

See the Cylc Flow PR here: https://github.com/cylc/cylc-flow/pull/3583/files

Reply:
Added, let me know if it should be moved up or down

Reply:
Looks fine to me.